config/ecs/otel-instance-metrics-config.yaml

```diff
@@ -1,3 +1,8 @@
+extensions:
+  health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
```

Comment:
This config still has an endpoint and path defined. Do we want to standardize on whether or not we specify them or not in configs?

Reply:
oh no. left there by mistake. standardized to not required in configs. Thanks, updated now. Checked other configs as well. They should not have this endpoint and path.